addons/base_import/static/csv/External_id_3rd_party_application_product_categories.csv

```diff
@@ -1,6 +1,6 @@
 External ID,Name,Parent Category/External ID
-a1,Expenses,product.product_category_all
-a2,Other Products,product.product_category_all
-a3,Sellable Products,product.product_category_all
+a1,Expenses,product.product_category_consumable
+a2,Other Products,product.product_category_consumable
+a3,Sellable Products,product.product_category_consumable

```

Comment:
No it will not work,
As we have to set this column for all the rows or none and since this is only for documentation to display how import work and `goods`  category seems best option in data